### ISSUE TITLE
a11y: Add focus outline to "Get this book" and "Table of Contents" buttons (& others)

### ISSUE THREAD

**Issue by ambersdai** [openstax/cnx#119]
## Overview
From https://drive.google.com/file/d/0BxiTbl6K_ZNRelpxRlkwYkZnb3c/view

Issue 9: Buttons have no focus or focus is not visible (color issue)
Buttons have no focus or focus is not visible (2.4.7, 1.3.1)
Example (the button with no focus border and bad focus visual change):

Example (button looked like a link):
https://drive.google.com/file/d/0B2fFZC6BqvTLTVJkUXdRUmN1UVk/view?usp=sharing

### Reference to standards
2.4.7 Focus Visible: Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible. (Level AA)
1.3.1 Info and Relationships: Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text. (Level A)
### Suggestion
One common way is a caret within the text field to indicate that the text field has the keyboard focus. 
Another way is a visual change of a button to indicate that that button has the keyboard focus.
## PRs
What PRs need to be merged to complete this task/feature
## Test Plan
How will QA know this feature works? What parts of regression tests would be most affected?
## Acceptance Criteria
Expected behavior

![](https://trello-attachments.s3.amazonaws.com/581c83e5cae45416fa9bf5e2/5a7df67b8486cf8b9fe47b5d/9af0e2a07e746aeec02c43d21080a2f6/image.png)

![](https://trello-attachments.s3.amazonaws.com/581c83e5cae45416fa9bf5e2/5a7df67b8486cf8b9fe47b5d/ef1b8d504ab563d36a38b17d06e0318e/image.png)

**Comment by ambersdai** [comment]
We could modify this card so that the colors reverse on the Contents button when selected. This is what we did with the Get This Book button: [Get This Book button should change appearance on focus](https://github.com/Connexions/webview/issues/1823)
